Sources/CAudioKit/AudioKitCore/Synth/CoreSynth.cpp

```diff
@@ -107,7 +127,7 @@ int CoreSynth::init(double sampleRate)
     data->segParameters[0].finalLevel = 0.2f;
     data->segParameters[0].seconds = 0.01f;
     data->segParameters[1].initialLevel = 0.2f;   // hold at 0.2 for 1 sec
-    data->segParameters[1].finalLevel = 0.2;
+    data->segParameters[1].finalLevel = 0.2F;
```

Comment:
This is minor but I've always seen float designated with a lowercase f, and all the other nearby values also have lowercase. There a few these added in this pr.

Reply:
Will fix. Just a matter of coding style, really. All other code bases I've used stick to upper case. Doesn't matter to me!